test/require.go

```diff
@@ -0,0 +1,62 @@
+package test
+
+import (
+	"os"
+	"strconv"
+	"testing"
+)
+
+const (
+	// UnitTest refers to the name of the environment variable that is used to
+	// specify that unit tests shall be run. Unless this environment variable
+	// is explicitly set to evaluate to false ("0", "no", or "false"), unit
+	// tests are executed all the time.
+	UnitTest = "AUTH_RESOURCE_UNIT_TEST"
+	// Database refers to the name of the environment variable that is used to
+	// specify that test can be run that require a database.
+	Database = "AUTH_RESOURCE_DATABASE"
+	// Remote refers to the name of the environment variable that is used to
+	// specify that test can be run that require availability of some remote servers such as Keycloak.
+	Remote = "AUTH_RESOURCE_REMOTE"
```

Comment:
I don't think we need this mode. Let's drop `Remote`.

Reply:
Removed.